ci-operator/jobs/operator-framework/operator-lifecycle-manager/operator-framework-operator-lifecycle-manager-master-periodics.yaml

```diff
@@ -0,0 +1,67 @@
+periodics:
+- agent: kubernetes
+  cron: 0 6 * * *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: master
+    org: operator-framework
+    repo: operator-lifecycle-manager
+  name: periodic-ci-operator-framework-operator-lifecycle-managment-master-rhoperator-metric-e2e-aws-olm-master-daily
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --branch=master
+      - --give-pr-author-access-to-namespace=true
+      - --lease-server=http://boskos
+      - --org=operator-framework
+      - --repo=operator-lifecycle-manager
+      - --resolver-address=http://ci-operator-configresolver-ci.svc.ci.openshift.org
+      - --secret-dir=/usr/local/e2e-aws-olm-cluster-profile
+      - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
+      - --target=e2e-aws-olm
+      - --template=/usr/local/e2e-aws-olm
+      command:
+      - ci-operator
+      env:
+      - name: CLUSTER_TYPE
+        value: aws
+      - name: CONFIG_SPEC
+        valueFrom:
+          configMapKeyRef:
+            key: operator-framework-operator-lifecycle-manager-master.yaml
+            name: ci-operator-master-configs
+      - name: JOB_NAME_SAFE
+        value: e2e-aws-operator-metric-daily
+      - name: TEST_COMMAND
+        value: make e2e-operator-metrics
+      image: ci-operator:latest
+      imagePullPolicy: Always
+      name: ""
+      resources:
+        requests:
+          cpu: 10m
+      volumeMounts:
+      - mountPath: /usr/local/e2e-aws-olm-cluster-profile
```

Comment:
A little weird that these are named after the other job but afaik that should be fine